CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.53.0
+------------------------------
+*August 1, 2018*
+
+### Changed
+ - height unit on fullScreenOverlay
```

Comment:
Always good to explain why – was this just a bug?  If so, this should be under `fixed` heading